jupyterhub_singleuser_profiles/images.py

```diff
@@ -111,15 +115,15 @@ def load(self):
                                             ),
                                             name=tag.name,
                                             recommended=tag_annotations.get(RECOMMENDED_ANNOTATION, False),
-                                            build_status=self.get_image_build_status(imagestream_name)
+                                            build_status=self.get_image_build_status(imagestream_name),
+                                            default=bool(strtobool(annotations.get(DEFAULT_IMAGE_ANNOTATION, "False")))
```

Comment:
If this has been moved to tag information, maybe it should use tag_annotations instead.

Reply:
Good catch!